src/Form/LegacyConsumer/templates/checkbox.html.php

```diff
@@ -0,0 +1,12 @@
+<input
+	type="checkbox"
+	class="give-input required"
+	name="give_<?php echo $field->getName(); ?>"
+	id="give-<?php echo $field->getName(); ?>"
+	<?php if ( $field->isRequired() ) : ?>    
+		required="" aria-required="true"
+	<?php endif; ?>
+	tabindex="1"
```

Comment:
@kjohnson is there any reason to hardcode tab index to `1` in all `*.html.php`

Reply:
@ravinderk yes, it is added for accessibility.